easybuild/framework/easyconfig/tweak.py

```diff
@@ -158,13 +158,15 @@ def tweak(easyconfigs, build_specs, modtool, targetdirs=None):
                 path = robot_find_easyconfig(source_toolchain['name'], source_toolchain['version'])
                 toolchain_ec = process_easyconfig(path)
                 toolchain_deps = resolve_dependencies(toolchain_ec, modtool, retain_all_deps=True)
-                toolchain_dep_specs = [dep['spec'] for dep in toolchain_deps if
-                                       dep['spec'] not in listed_ec_paths]
+                toolchain_dep_filenames = [os.path.basename(dep['spec']) for dep in toolchain_deps]
+                # only retain toolchain dependencies that are not in original list of easyconfigs to tweak
```

Comment:
`that are` or `that appear in` rather than `that are not`

Reply:
No, the condition says `not in`, so the comment is correct to say `retain ... are not in list of easyconfigs`?

Reply:
Yeah, sorry, been a while since I looked at this